definition/method/method_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2018 Caicloud Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package method
+
+import (
+	"reflect"
+	"testing"
+)
+
+type TestInterface interface {
+	Number() int
+}
+
+type TestStruct struct {
+	a int
+}
+
+func (t *TestStruct) Number() int {
+	return t.a
+}
+
```

Comment:
add another struct implement TestInterface，used for test putInterface.

+func (t *Test2Struct) Number() int {
+	return t.a*2
+}